src/w3bstream_client_python/client.py

```diff
@@ -17,18 +17,17 @@ def __init__(self, device_id: str, event_type: str = 'DEFAULT', timestamp: datet
 
 
 PUBLISH_INTERVAL = 5
-PUBLISH_BATCH_SIZE = 10
+PUBLISH_BATCH_SIZE = 1
```

Comment:
modified

Reply:
Currently, publish async only works well if `PUBLISH_BATCH_SIZE = 1`.  It is recommended to rename it as private, such as `_PUBLISH_BATCH_SIZE`, indicating that it should not be modified from outside the module.